test/fixtures/privileges.sql

```diff
@@ -17,6 +17,7 @@ GRANT ALL ON TABLE
     , comments
     , complex_items
     , compound_pk
+    , empty_table
```

Comment:
The previous 404/401 subtlety had even caused a problem with a test which was getting 404 because of an access denied rather than directly as a result of there being no data.